src/Routes/ImageManager/ImageSetsTable.js

```diff
@@ -175,15 +182,15 @@ const ImageTable = ({
       ) : (
         <GeneralTable
           apiFilterSort={true}
-          historyProp={historyProp}
-          locationProp={locationProp}
-          navigateProp={navigateProp}
+          //  historyProp={historyProp}
```

Comment:
Those properties cannot be removed once we are using on inventory side.
The context and props can coexist.
Thougths?

Reply:
Context and Props override each other because both of them modifying the general table